tests/services/label_data_parsing/parsing/test_resp_parsing.py

```diff
@@ -959,13 +959,166 @@ def test_video_project_classification():
 
     parsed_label = ParsedLabel(label=label, json_interface=json_interface, input_type="VIDEO")
 
-    # assert parsed_label.frames[5].jobs["FRAME_CLASSIF_JOB"].is_key_frame is True
-    # assert parsed_label.frames[5].jobs["FRAME_CLASSIF_JOB"].category.name == "OBJECT_A"
+    assert len(parsed_label.frames) == 9
+    assert not hasattr(parsed_label, "jobs")
+
+    assert parsed_label.frames[5].jobs["FRAME_CLASSIF_JOB"].is_key_frame is True
```

Comment:
If `frames` is a dict, there can be missing keys as well? like 0 to 4 for example

Reply:
frames is a list of ParsedJobs